apps/schema/src/components/ColumnEdit.vue

```diff
@@ -37,7 +37,9 @@
         v-if="
           column.columnType == 'REF' ||
           column.columnType == 'REF_ARRAY' ||
-          column.columnType == 'REFBACK'
+          column.columnType == 'REFBACK' ||
+          column.columnType == 'ONTOLOGY' ||
```

Comment:
this is actually bugfix to allow schema editor to change type to ONTOLOGY or ONTOLOGY_REF. The upload already did this.